portability/toku_instr_mysql.h

```diff
@@ -12,8 +12,19 @@
 // undefine them here to avoid compilation errors.
 #undef __STDC_FORMAT_MACROS
 #undef __STDC_LIMIT_MACROS
-#include <mysql/psi/mysql_file.h>    // PSI_file
-#include <mysql/psi/mysql_thread.h>  // PSI_mutex
+#include "mysql/psi/mysql_file.h"    // PSI_file
+#include "mysql/psi/mysql_thread.h"  // PSI_mutex
+#include "mysql/psi/mysql_stage.h"   // PSI_stage
+
+#if (MYSQL_VERSION_ID >= 80000)
+#include "mysql/psi/mysql_cond.h"
+#include "mysql/psi/mysql_mutex.h"
+#include "mysql/psi/mysql_rwlock.h"
+
+#elif (MYSQL_VERSION_ID >= 50700)
+#include "mysql/psi/mysql_memory.h"
```

Comment:
It looks now mysql_memory.h will not be included in 8.0? The checks against 8.0 and 5.7 have to be independent

Reply:
Yeah, you're correct.  Weird, it passed all build tests, so perhaps mysql_memory.h isn't even needed here...

Reply:
OK, so mysql_memory.h isn't required at all.  This makes some sense as the FT PFS implementation was initially done against 5.6 which did not have memory instrumentation and we never went back to add it in 5.7.  Fixing...

Reply:
Pushed new changes but no new jenkins as jenkins seems unable to schedule new jobs at the moment.  It did pass compilation on 5.6, 5.7, and 8.0.12 locally :-\